src/backend/web/app.js

```diff
@@ -9,4 +9,5 @@ app.use('/health', healthcheck());
 
 app.use('/', router);
 
+
```

Comment:
Why the newline here?

Reply:
Yeah I messed up and removed the newline. It was an issue from a different branch.